modules/bmc/bmc_plugin.rb

```diff
@@ -5,5 +5,16 @@ class Plugin < Proxy::Plugin
     default_settings :redfish_verify_ssl => true
     validate :redfish_verify_ssl, :boolean => true
     plugin :bmc, ::Proxy::VERSION
+
+    # Various installed providers are exposed as capabilties
+    capability 'redfish'
+    capability 'shell'
+    capability 'ssh'
+    capability -> { Proxy::BMC::IPMI.providers_installed }
```

Comment:
Please extract non-ipmi providers into a constant and use that here and also in the `non_ipmi_providers` method (or elsewhere when appropriate) so there's only one place to edit.

Reply:
What would be a good place to store it? `Proxy::BMC::IPMI.non_ipmi_providers` doesn't make sense.